dlp/synth.py

```diff
@@ -47,8 +50,8 @@
 s.replace(
     "google/cloud/dlp_v2/proto/storage_pb2.py",
     "number regex.*\n(\s+)latex:.*\n",
-    "number regex \"(\\d\{3\}) \\d\{3\}-\\d\{4\} \"\\\n"
-    "\g<1>could be adjusted upwards if the area code is \\\n"
+    'number regex "(\\d\{3\}) \\d\{3\}-\\d\{4\} "\\\n'
+    "\g<1>could be adjusted upwards if the area code is \\\n",
```

Comment:
Does black allow that?  It is the spelling I would prefer, but I thought it didn't take.